src/pages/Payment.vue

```diff
@@ -90,7 +86,7 @@ export default {
     },
     methods: {
         showConfirmation: function() {
-            this.setMessage("Thanks, your payment request has been sent.", constants.MESSAGE_SUCCESS);
+            this.setMessage("Thanks, your payment request has been sent. Please take your vouchers to your market representative for them to be sent off.", constants.MESSAGE_SUCCESS);
```

Comment:
Yes, I think it will need to be a variable - but I believe that's part of the card @ob6160 is working on, so I thought we could merge this in and then he could replace the static text with a variable.